media/js/src/simulationOne/simulationOne.jsx

```diff
@@ -199,43 +199,58 @@ export const SimulationOne = () => {
                                 <div className='simulation__step-content'>
-                                    <div className='row'>
+                                    <p>
+                                    The graph generated with your sample size
+                                    and correlation coefficient produces the
+                                    following coefficients. Let&rsquo;s look at
+                                    them in detail.
+                                    </p>
+                                    <h2 className='mt-4'>
+                                        Regression line equation:</h2>
+                                    <div className='ms-3 mb-3'>
                                         <Katex tex={
-                                            // eslint-disable-next-line max-len
-                                            'y = \\hat{\\beta_0} + \\hat{\\beta_1}x'
+                                        // eslint-disable-next-line max-len
+                                            '\\hat{y} = \\hat{\\beta_0} + \\hat{\\beta_1}x'
                                         } />
                                     </div>
-                                    <div className='row'>
+                                    <div className='ms-3 mb-3'>
                                         <Katex tex={
                                         // eslint-disable-next-line max-len
-                                            `y = ${intercept.toFixed(2)} + ${slope.toFixed(3)}x`
+                                            `\\hat{y} = ${intercept.toFixed(2)} + ${slope.toFixed(3)}x`
                                         } />
                                     </div>
-                                    <div className='row'>
-                                        <label>
-                                            <Katex tex={
-                                            // eslint-disable-next-line max-len
-                                                `\\hat{\\beta_0} = ${intercept.toFixed(3)}`
-                                            } />
-                                        </label>
+                                    <p>where <span className='katex'>
+                                        <span className='mathnormal'>y</span> {/* eslint-disable-line max-len */}
+                                    </span> is the dependent variable and
+                                    <span className='katex'> <span className='mathnormal'>x</span> {/* eslint-disable-line max-len */}
+                                    </span> is the independent, or
+                                        explanatory, variable.
+                                    </p>
+                                    <h2 className='mt-4'>Sample <span className='katex'> <span className='mathnormal'>y</span> {/* eslint-disable-line max-len */}
+                                    </span>-intercept coefficient:</h2>
```

Comment:
I recommend treating `y-intercept` as a unit. Either all within Katex or without Katex since it is also represented by the beta_0.

Reply:
Good point. I will fix